xml/ulp.xml

```diff
@@ -120,8 +113,8 @@
         <package>glibc-livepatches</package> and
```

Comment:
I suggest to rephrase to: 

`To receive <systemitem>glibc</systemitem> and <systemitem>openssl</systemitem> live patches, install the following packages: `

(because the specific packages are mentioned in the` <screen>` below anyway)